packages/block-library/src/navigation-link/edit.js

```diff
@@ -304,28 +296,41 @@ function NavigationLinkEdit( {
 	);
 }
 
+// function getRGB
 export default compose( [
 	withSelect( ( select, ownProps ) => {
 		const {
 			getBlockAttributes,
 			getBlockParents,
 			getClientIdsOfDescendants,
 			hasSelectedInnerBlock,
+			getColorObjectByColorSlug,
 		} = select( 'core/block-editor' );
 		const { clientId } = ownProps;
 		const rootBlock = getBlockParents( clientId )[ 0 ];
-		const navigationBlockAttributes = getBlockAttributes( rootBlock );
+		const navBlockAttrs = getBlockAttributes( rootBlock );
 		const hasDescendants = !! getClientIdsOfDescendants( [ clientId ] )
 			.length;
 		const showSubmenuIcon =
-			!! navigationBlockAttributes.showSubmenuIcon && hasDescendants;
+			!! navBlockAttrs.showSubmenuIcon && hasDescendants;
 		const isParentOfSelectedBlock = hasSelectedInnerBlock( clientId, true );
```

Comment:
Minor: in reality - due to the the 3rd `true` arg triggering "deep" mode - this [`isParentOfSelectedBlock `] is actually determining if it's an _ancestor_ of the selected Block.